package.json

```diff
@@ -70,5 +74,9 @@
     "/docs/_shared",
     "/e2e/",
     "**/build"
-  ]
+  ],
+  "resolutions": {
+    "**/@babel/core": "^7.9.6",
+    "**/webpack": "^4.43.0"
```

Comment:
Technically `next` should declare it as a peer dependency with a default but that isn't supported in most package managers. In yarn v2 we will leverage `packageExtensions` instead of `resolutions`